extensions/jsonb/src/main/java/io/jsonwebtoken/jsonb/io/JsonbDeserializer.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2014 jsonwebtoken.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.jsonwebtoken.jsonb.io;
+
+import io.jsonwebtoken.io.DeserializationException;
+import io.jsonwebtoken.io.Deserializer;
+
+import javax.json.bind.Jsonb;
+import javax.json.bind.JsonbException;
+import java.nio.charset.StandardCharsets;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * @since 0.10.0
```

Comment:
This would need to be 0.12 or higher.

Actually, you can use `@since JJWT_RELEASE_VERSION` and we will set the version before the release 😄